<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickBinSingleDialogBase</class>
 <widget class="QDialog" name="PickBinSingleDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick Binnings from list...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Pick your list of binnings...or hold the 'Ctrl' key for multiple selections, or hold the 'Shift' key for a range selection.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPushButton" name="PushButtonOk">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Bins available</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="mUi_TreeWidget">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Pick your bins from the list...</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>25</number>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Bin#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bin name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
